clmm/theory/ccl.py

```diff
@@ -88,9 +88,6 @@ def _update_halo_density_profile(self):
         """"updates halo density profile with set internal properties"""
         # prepare mdef object
         self.mdef = ccl.halos.MassDef(self.delta_mdef, self.mdef_dict[self.massdef])
```

Comment:
Let me see if I got this correctly. We only need this line with `2.6.2dev7<CCL<2.7`, and with `CCL>2.7` `_set_concentration` does not work anymore?